src/esp/bindings/SimBindings.cpp

```diff
@@ -72,6 +77,9 @@ void initSimBindings(py::module& m) {
             Not available for all datasets
             )")
       .def_property_readonly("renderer", &Simulator::getRenderer)
+      .def_property_readonly(
+          "gfx_replay", &Simulator::getGfxReplayManager,
```

Comment:
```suggestion
          "gfx_replay_manager", &Simulator::getGfxReplayManager,
```

to be consistent with the C++ method?